src/features/confirm/components/ConfirmButton.tsx

```diff
@@ -45,6 +45,9 @@ export const ConfirmButton = (props: Omit<BaseButtonProps, 'onClick'> & { id: st
             }
           }
         })
+        .catch((error) => {
+          dispatch(ProcessActions.completeRejected({ error: JSON.parse(JSON.stringify(error)) }));
+        })
```

Comment:
Is it possible to use `structuredClone()` instead?

```suggestion
        .catch((error) => {
          dispatch(ProcessActions.completeRejected({ error: structuredClone(error) }));
        })
```

Reply:
Unfortunately structuredClone does not seem to work for AxiosErrors for some reason: https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm#:~:text=TypedArray-,Error%20types,-For%20Error%20types